zerver/tests/test_upload.py

```diff
@@ -538,6 +540,19 @@ def test_file_download_authorization_public(self) -> None:
             self.assertEqual(b"zulip!", data)
             self.logout()
 
+    def test_serve_local(self) -> None:
+        with self.settings(SENDFILE_BACKEND='sendfile.backends.nginx'):
+            _get_sendfile.clear()  # To clearout cached version of backend from djangosendfile
+            filename = os.path.join(settings.LOCAL_UPLOADS_DIR, 'files', 'á.jpg')
+            basedir = os.path.dirname(filename)
+            if not os.path.exists(basedir):
+                os.makedirs(basedir)
+            open(filename, 'w').close()
+            responce = serve_local(HttpRequest(), 'á.jpg')
+            _get_sendfile.clear()
+            test_upload_dir = os.path.split(settings.LOCAL_UPLOADS_DIR)[1]
+            self.assertEqual(responce['X-Accel-Redirect'], '/serve_uploads/../../' + test_upload_dir + '/files/%C3%A1.jpg')
```

Comment:
`response` is mispelled.

Reply:
will fix, I should probably get a spell check plugin